packages/cli/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.build.json",
   "include": ["src"],
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
Is this needed? If so, we should flag what dependency causes the issue and keep track of it.

Reply:
This is needed. I'm not sure when it became a problem, but without enabling `esModuleInterop` the following imports fail: 
- `chalk`
- `ora`
- `cli-table3`

Some can be resolved by making them * imports, however cli-table3 remains a problem even converted to an * import. Based off of [the TypeScript docs](https://www.typescriptlang.org/tsconfig#esModuleInterop) it seems to be a problem with CommonJS modules not strictly conforming to Typescript.